bedrock/firefox/templates/firefox/new/onboarding/scene1.html

```diff
@@ -85,7 +85,21 @@
                 <li><a class="more" href="{{ url('firefox.family.index') }}">{{_('Discover all the ways to take Firefox on the go')}}</a></li>
               </ul>
             </div>
-            <button id="other-platforms-modal-link" class="hidden" type="button">{{_('Firefox for Other Platforms & Languages')}}</button>
+
+            <div id="other-platforms-languages-wrapper" class="hidden">
+            {% if l10n_has_tag('firefox_new_install_options') %}
+              <button id="other-platforms-modal-link" type="button">
```

Comment:
This is a nit and not something I expect to be solved in this PR (I'm not sure of the right answer), but from a UX perspective it feels a bit weird for the first link to open a modal and then one directly underneath act like a normal link. It feels like this element should be styled more like a button. Maybe something for Rachel to think about when the redesign happens?